### PR TITLE
Option to send more information about navigation issues

### PR DESCRIPTION
**Pull request by ironsidevsquincy**
Also make it more extendable, in case other drop down elements need to send other data

### CODE REVIEW

**Comment by keirog** [comment]
FYI, will need some changes to [next-user-feedback](https://github.com/Financial-Times/next-user-feedback/blob/master/server/routes/opt-out.js#L13)

**Comment by keirog** [comment]
Also https://github.com/Financial-Times/next-opt-out-interstitial/issues/39 in case it's relevant

**Comment by ironsidevsquincy** [comment]
https://github.com/Financial-Times/next-user-feedback/pull/23

**Comment by keirog** [comment]
:+1:

File: client/js/more-toggler.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+/*jshint node:true*/
+/*global document*/
```
Comment (matthew-andrews) [inline client/js/more-toggler.js]: as you're using `nbt verify` shouldn't need line 1 & 2 here

**Pull request merged**